conda_forge_tick/migrators/migration_yaml.py

```diff
@@ -492,9 +492,10 @@ def create_rebuild_graph(
         )
         # get host/build, run and test and launder them through outputs
         # this should fix outputs related issues (eg gdal)
-        all_reqs = (host or build) | requirements.get("run", set())
+        all_reqs = requirements.get("run", set())
```

Comment:
This is populated with `run_exports` from `host` ?

Reply:
In an approximate way, yes: https://github.com/regro/cf-scripts/blob/master/conda_forge_tick/make_graph.py#L172

The approximation made in the code in the link is not different from keeping host/build sections. Thus I think this should be ok.

Reply:
The code there is not perfect so we might hold off on this change for a while.

Reply:
Actually no, let's ship this now and see what happens.